data/card_catalogue/social.json

```diff
@@ -57,7 +57,7 @@
     "tier": "blue",
     "details": {
       "rules": {
-        "active": "Speak with any living NPC, then draw another blue.",
+        "active": "Speak with any living NPC. They will be unaware of who they're speaking to.",
```

Comment:
As previously discussed, stripping out the card drawing clause and making this a little more distracting to bring it up to an acceptable power level.